crypto/armor.go

```diff
@@ -207,3 +209,36 @@ func decryptPrivKey(saltBytes []byte, encBytes []byte, passphrase string) (privK
 
 	return legacy.PrivKeyFromBytes(privKeyBytes)
 }
+
+//-----------------------------------------------------------------
+// encode/decode with armor
+
+func EncodeArmor(blockType string, headers map[string]string, data []byte) string {
+	buf := new(bytes.Buffer)
+	w, err := armor.Encode(buf, blockType, headers)
```

Comment:
All this is copy/pasted from Tendermint?

Reply:
yes, this was removed from Tendermint. It wasn't used there.